src/AVFoundation/Enums.cs

```diff
@@ -393,6 +403,10 @@ public enum AVAudioSessionCategoryOptions : ulong {
 #endif
 		[NoWatch, iOS (10,0), TV (10,0)]
 		AllowAirPlay = 64,
+		[NoMac]
+		[NoTV]
+		[iOS (14,5), Watch (7,3)]
+		OverrideMutedMicrophoneInterruption = 128
```

Comment:
```suggestion
		OverrideMutedMicrophoneInterruption = 128,
```
Smaller diff next time

Reply:
right! thanks for catching these